### DISCUSSION TITLE
Send mute state via MQTT

### DISCUSSION THREAD

**Discussion by mill1000** [mikebrady/shairport-sync#1646] [Ideas]
Would it be possible for the MQTT interface to send the state of the audio mute?

Via MQTT we can toggle mute, but currently there's no way of knowing if the player was just muted or unmuted.

**Comment by mikebrady** [comment]
Thanks. Let me take a look at this. Does MQTT report the volume? If so, does it report `-144.0` when audio is muted?

**Reply by mill1000** [reply]
MQTT does not report volume upon entering mute, but does report it when exiting mute.

```shell
0.000092656 "audio_alsa.c:2050" mute(1) set_mute_state
0.000030730 "player.c:3218" player_volume_without_notification: volume mode is 1, airplay_volume is -144.000000, software mute is enabled.
// trimmed
0.96927880 "rtsp.c:3556" Connection 1: request to set AirPlay Volume to: 0.000000.
0.001089687 "audio_alsa.c:2050" mute(0) set_mute_state
0.000067761 "mqtt.c:108" [MQTT]: publishing under stereoberry/shairport/volume
0.000398385 "player.c:3356" player_volume_without_notification: volume mode is 1, airplay volume is 0.00, software_attenuation dB: 0.00, hardware_attenuation dB: 0.00, muting is disabled.
```

**Reply by mill1000** [reply]
Hey @mikebrady. Just want to confirm if you saw the above reply that answered your question.

**Comment by mikebrady** [comment]
Oops -- apologies; I didn't see this, I'm afraid. Let me take a look at it. Apologies again.
